lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/client/LoggingJobStatusHandler.java

```diff
@@ -28,15 +28,25 @@
 /**
  * A handler for job status messages that just logs them.
  */
-final class LoggingJobStatusHandler implements EventHandler<ReefServiceProtos.JobStatusProto> {
+@DriverSide
+public class LoggingJobStatusHandler implements JobStatusHandler {
+
   private static final Logger LOG = Logger.getLogger(LoggingJobStatusHandler.class.getName());
 
+  private ReefServiceProtos.JobStatusProto lastStatus = null;
+
   @Inject
   LoggingJobStatusHandler() {
   }
 
   @Override
   public void onNext(final ReefServiceProtos.JobStatusProto jobStatusProto) {
-    LOG.log(Level.INFO, "Received a JobStatus message that can't be sent:\n" + jobStatusProto.toString());
+    this.lastStatus = jobStatusProto;
+    LOG.log(Level.INFO, "In-process JobStatus:\n{0}", jobStatusProto);
```

Comment:
@jwang98052, do we have any test code that relies on this particular log message?